builtin/providers/aws/utils.go

```diff
@@ -0,0 +1,13 @@
+package aws
```

Comment:
Rather than a generic "utils" file, maybe this file should be named base64.go and likewise for the tests?